cfgov/jinja2/v1/index.html

```diff
@@ -5,8 +5,8 @@
 {%- endblock %}
 
 {% block desc -%}
-    Our vision is a consumer finance marketplace that works for American consumers,
-    responsible providers, and the economy as a whole.
+    {{ 'Our vision is a consumer finance marketplace that works for American consumers, ' +
+       'responsible providers, and the economy as a whole.' }}
```

Comment:
Are we that allergic to letting a line run long that we want to go to these lengths? I'm a big proponent of semantic linebreaks, and I still wouldn't go this far :)